CONTRIBUTING.rst

```diff
@@ -18,13 +19,7 @@ Accordingly, the `repository <https://github.com/rucio/rucio/>`_  consists of th
  * the **next** branch includes the development for the next major version.
  * the **hotfix** branch includes the patch for hotfix releases.
 
-Thus, on release day of a feature release both master and next are the same,
-afterwards they diverge until the next feature release.
-Pull requests for **features** are only made against the **next** branch.
-Pull requests for **patches** are made against the **next** and **master** branch, as
-these bugfixes need to be represented in both branches. Thus two
-pull requests are needed for patches, and the helper scripts do it
-automatically for you.
+Thus, on release day of a feature release both master and next are the same, afterwards they diverge until the next feature release. Pull requests for **features** are only made against the **next** branch. Pull requests for **patches** are made against the **next** and **master** branch, as these bugfixes need to be represented in both branches. Thus two pull requests are needed for patches, and the helper scripts do it automatically for you.
```

Comment:
You don't mention the hotfix

Reply:
I just removed the useless newlines ;-) I'll add it.